packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -190,6 +563,181 @@ export const logExperimentUpdated = async ({
   return emittedEvent.id;
 };
 
+/**
+ * Deletes an experiment by ID and logs the event for the organization
+ * @param id
+ * @param organization
+ */
+export async function deleteExperimentByIdForOrganization(
+  id: string,
+  organization: OrganizationInterface
+) {
+  try {
+    const previous = await findExperiment({
+      experimentId: id,
+      organizationId: organization.id,
+    });
+    if (previous) {
```

Comment:
Previous needs to run through the toInterface method

Reply:
Now that we've updated this to take in the experiment that is returned from `getExperimentById` rather than getting it again, it should now already be the ExperimentInterface rather than the ExperimentDocument.